src/main/java/jalau/at18/azul/view/AzulFrame.java

```diff
@@ -8,11 +8,20 @@ public class AzulFrame extends JFrame {
     public static final int INITIAL_POSITION = 200;
     public static final int WIDTH = 900;
     public static final int HEIGHT = 600;
-
+    public static final int WIDTH1 = 50;
```

Comment:
We could change the name of some constants to make their use more clear, for example instead of `WIDTH1` it could be `TILES_WIDTH`, please do the same for the constants that don't have a clear name

Reply:
already change the width1 and height1 to tile_width and tile_height